### ISSUE TITLE
How to run csx file?

### ISSUE THREAD

**Issue by Elufimov** [OmniSharp/omnisharp-vscode#1333]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.1)

Product Information:
 Version:            1.0.1
 Commit SHA-1 hash:  005db40cd1

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.1
```
VS Code version: `1.10.2`
C# Extension version: `1.8.0`

Hi! I am want to try using vs code as lite scripting ide. However I can not find out how to run csx file. Is there a some documentation about it? My current steps are:
1) Create project `dotnet new console`
2) Open folder in vsc
3) Restore the packages
4) Add file `test.csx`
And then I have autocomplete and intellisense inside this file.

**Comment by DustinCampbell** [comment]
@filipw?

**Comment by filipw** [comment]
Hi @Elufimov 

So to run a CSX file you need a CSX runner on your machine - kinda like to run a .NET Core application, you need a .NET Core installation or to run a desktop .NET application, you need desktop .NET installation (plus possibly Mono).

The most popular CSX runners out there are:
 - CSI (the official Roslyn script runner). It ships with MSBuild tools so it's likely already on your machine. Here is [an article](https://msdn.microsoft.com/en-us/magazine/mt614271.aspx) to help you get started. CSI works on windows + on Mono (no .NET Core)
 - [scriptcs](https://github.com/scriptcs/scriptcs) is an open source project that extends a bit the abilities of CSI (works on windows + on Mono (no .NET Core))
 - [dotnet script](https://github.com/filipw/dotnet-script) is another open source project that allows running scripts using .NET Core (so specifically targeting the gap other two don't support)

So in other words, Omnisharp and VS Code in this case only support intellisense, language services and refactoring capabilities for CSX, but you need a runner separately.

Hope this helps.

@DustinCampbell I think since this is information-only issue, this can be closed now.

**Comment by Elufimov** [comment]
@filipw Thx for the clarification! This really helped me.
